docker-compose.yml

```diff
@@ -15,3 +15,4 @@ services:
       - GAS_LIMIT=${GAS_LIMIT:-10000000}
       - STAKE_TOKEN=${STAKE_TOKEN:-ujunox}
       - TIMEOUT_COMMIT=${TIMEOUT_COMMIT:-5s}
+      - UNSAFE_CORS=true
```

Comment:
I guess we'll need to remember to disable UNSAFE_CORS before merging into a release, right?